netbout-web/src/main/java/com/netbout/dynamo/DyAlias.java

```diff
@@ -58,7 +58,7 @@
     /**
      * Valid email pattern.
      */
-    private static final Pattern MAIL;
+    private static Pattern mail;
```

Comment:
@mkordas If I let it how it was it gave me chekstyle  violation for not using DyAlias.MAIL , if I use DyAlias.MAIL it says can not assign value to final variable MAIL. so i removed the final and  there are no violations now.